static/js/common.js

```diff
@@ -1,3 +1,8 @@
+// This reloads the module in development rather than refreshing the page
+if (module.hot) {
```

Comment:
Yeah, we could just remove this from the files requiring coverage.  Or add whatever comment tags a line as not needing coverage with istanbul, since we probably don't care about testing this.